src/app/core/user.service.ts

```diff
@@ -242,6 +242,28 @@ export class UserService {
     return deleteValue;
   }
 
+  combineName(firstname: string, lastname: string){
+    let firstnameArray = firstname.split(' ');
+    let lastnameArray = lastname.split(' ');
```

Comment:
The names might consist trailing spaces and we can remove them with `trim()` so that we don't have empty strings in our array.
```suggestion
    let firstnameArray = firstname.trim().split(' ');
    let lastnameArray = lastname.trim().split(' ');
```